things/postgres/channels.go

```diff
@@ -137,13 +137,16 @@ func (cr channelRepository) RetrieveAll(ctx context.Context, owner string, pm th
 	if nq != "" {
 		query = append(query, nq)
 	}
+	if ownerQuery != "" {
+		query = append(query, ownerQuery)
+	}
 
 	if len(query) > 0 {
-		whereClause = fmt.Sprintf("AND %s", strings.Join(query, " AND "))
+		whereClause = fmt.Sprintf(" WHERE %s", strings.Join(query, " AND "))
 	}
 
-	q := fmt.Sprintf(`SELECT id, name, metadata FROM channels
-	      WHERE owner = :owner %s ORDER BY %s %s LIMIT :limit OFFSET :offset;`, whereClause, oq, dq)
+	q := fmt.Sprintf(`SELECT id, name, metadata FROM channels 
+	%s ORDER BY %s %s LIMIT :limit OFFSET :offset;`, whereClause, oq, dq)
```

Comment:
Is this formatting OK? Looks like there is a trailing space at the end of the line, and line below is no indented (maybe this is OK, but previous was, and this probably makes it easier to read).

Reply:
Updated 👍